### ISSUE TITLE
Can i use Windows System library instead of Appium for WinAppDriver?

### ISSUE THREAD

**Issue by sheryK** [microsoft/WinAppDriver#1872]
I have built automated UI Regression Tests using WinAppDriver for Desktop app in my company. Developers dont like using Appium and thinking why adding Appium Server. They are pointing use of Windows System Library instead.
Any thought on that or if anyone has experience to share appreciate it.

**Comment by anunay1** [comment]
At least I am not aware of any such library. Ask them to implement themselves?

**Comment by sheryK** [comment]
it is System.Windows.Automation namespace done by Microsoft

**Comment by PandaMagnus** [comment]
Yes, you can. I've done it once or twice but it's a very different paradigm than what most testing libraries take. On the flip side, I seem to recall it was extremely customizable in how you can search for UI elements (it's been maybe 4 or so years since I dabbled in this.)

https://learn.microsoft.com/en-us/dotnet/framework/ui-automation/ui-automation-overview
